user-guide/Advanced_Modules/Data_Aggregator/Data_Aggregator_queries.md

```diff
@@ -34,6 +31,23 @@ To get a correctly configured query, you can make use of the DataMiner Dashboard
 
    with payload:
 
+   ## [From DataMiner 10.5.7/10.4.0-CU6/10.5.0-CU4 onwards](#tab/tabid-1)
+
+   > ``` json
+   > {
+   >   "connection": "", // copied in previous step
+   >   "options": { "Contract": 1 },
+   >   "query": {} // copied in previous step
+   > }
+   > ```
+
+   As value for the `Contract` property, specify:
+
+   * 0 = when GQI queries are processed by SLHelper
```

Comment:
Maybe clearer to say

 * Use `"Contract": 0` for queries that should be executed via CoreGateway and SLHelper.
 * Use `"Contract": 1` for queries that should be executed via the GQI DxM.
 
 See: ... _link to enabling GQI DxM setting_